File: services/quarkus-svc/src/main/java/com/perfsim/quarkussvc/domain/Order.java
package com.perfsim.quarkussvc.domain;

import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.time.OffsetDateTime;

@Entity
@Table(name = "orders", schema = "quarkus")
public class Order extends PanacheEntityBase {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;

    @Column(nullable = false)
    public String customer;

    @Column(nullable = false, length = 32)
    public String status = "PENDING";

    @Column(name = "total_cents", nullable = false)
    public long totalCents;

    @Column(name = "created_at", nullable = false, insertable = false, updatable = false)
    public OffsetDateTime createdAt;
}

File: services/symfony-svc/overlay/src/Entity/Order.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'orders')]
class Order
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'bigint')]
    public ?string $id = null;

    #[ORM\Column(type: 'string', nullable: true)]
    public ?string $customer = null;

    #[ORM\Column(type: 'string', nullable: true)]
    public ?string $status = null;

    #[ORM\Column(name: 'total_cents', type: 'bigint', nullable: true)]
    public ?string $totalCents = null;
}
